ui/pages/home/home.component.js

```diff
@@ -591,6 +596,29 @@ export default class Home extends PureComponent {
             }
           />
         ) : null}
+        {editedNetwork ? (
+          <ActionableMessage
+            type="success"
+            className="home__new-tokens-imported-notification"
```

Comment:
Is this class name correct?  Should it be new network instead of new tokens?

Reply:
It's because I'm using an existing CSS class; I don't want to create a new class.